Sentinel/Classes/Core/Internal/SentinelInternal.swift

```diff
@@ -15,9 +15,47 @@ extension Sentinel {
     ///     - tools: Tools which will be available in the Sentinel.
     ///     - viewController: The view controller from where will the Sentinel be presented.
     func present(tools: [Tool], on viewController: UIViewController) {
-        let items = tools.map { NavigationToolTableItem(title: $0.name, navigate: $0.presentPreview(from:)) }
-        let section = ToolTableSection(title: "Tools", items: items)
-        let toolTable = ToolTable(name: "Sentinel", sections: [section])
-        toolTable.presentPreview(from: viewController, push: false)
+        var tabBarControllers = createTabBarControllers(with: tools, viewController: viewController)
+        let toolsViewController = createToolsController(with: tools, viewController: viewController)
+        let deviceViewController = createDeviceViewController()
+
+        tabBarControllers.insert(deviceViewController, at: 0)
+        tabBarControllers.insert(toolsViewController, at: 2)
+
+        let tabBarController = UIStoryboard.sentinel.instantiateViewController(ofType: SentinelTabBarController.self)
+        tabBarController.setupViewControllers(with: tabBarControllers)
+        tabBarController.title = "Sentinel"
+
+        let navContoller = UINavigationController(rootViewController: tabBarController)
+        viewController.present(navContoller, animated: true)
+    }
+}
+
+private extension Sentinel {
+    func createTabBarControllers(with tools: [Tool], viewController: UIViewController) -> [UIViewController] {
+        return tools.filter { tool in
+            print(tool.name)
+            return tool.name == "Application" ||
+            tool.name == "Permissions" ||
+            tool.name == "Performance"
```

Comment:
I'm not a fan of this since it's error prone and needs to be updated if we update a tool name, especially since it's repeating in the function below as well. 

What I would do is add an enum property to `Tool` specifying it's type, and then use those enum values for filtering.